A2-2015/A2-Rewrite/frontend/package.json

```diff
@@ -3,11 +3,12 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "material-ui": "1.0.0-beta.37",
+    "ajv": "^6.3.0",
```

Comment:
I recommend not allowing yarn to add the ```^``` character because it will try to upgrade dependencies, instead, my recommendation is to specify exactly the needed version like ```"react": "16.2.0",```